packages/app-store/cal-ai/package.json

```diff
@@ -5,7 +5,9 @@
   "version": "0.0.0",
   "main": "./index.ts",
   "dependencies": {
-    "@calcom/lib": "*"
+    "@calcom/lib": "*",
+    "@calcom/prisma": "workspace:^",
```

Comment:
Once we remove the above cal-ai/lib/stripe.ts dependency, we can remove these deps I think.